resolver/resolver.go

```diff
@@ -29,18 +27,17 @@ var (
 
 // Resolver holds configuration state and the resource records
 type Resolver struct {
-	version    string
-	config     records.Config
-	rs         *records.RecordGenerator
-	rsLock     sync.RWMutex
-	leader     string
-	leaderLock sync.RWMutex
-	rng        *rand.Rand
+	masters []string
+	version string
+	config  records.Config
```

Comment:
Since `records.Config` already has a `Masters` field, why do we need another one here?